scripts/anonymize.sql

```diff
@@ -27,13 +27,62 @@ SET @common_hash_secret := rand();
 
 SET FOREIGN_KEY_CHECKS=0;
 
-TRUNCATE actioncounters_actioncounterunique;
+-- Unmodified Tables
+-- attachments_attachment
+-- attachments_attachmentrevision
+-- attachments_documentattachment
+-- auth_group
+-- auth_group_permissions
+-- auth_permission
+-- auth_user_groups
+-- auth_user_user_permissions
+-- django_content_type
+-- django_migrations
+-- django_site
+-- feeder_bundle
+-- feeder_bundle_feeds
+-- feeder_entry
+-- feeder_feed
+-- search_filter
+-- search_filtergroup
+-- search_index
+-- search_outdatedobject
+-- soapbox_message
+-- tagging_tag
+-- tagging_taggeditem
+-- taggit_tag
+-- taggit_taggeditem
+-- waffle_flag
+-- waffle_flag_groups
+-- waffle_flag_users
+-- waffle_sample
+-- waffle_switch
+-- wiki_document
+-- wiki_documentdeletionlog
+-- wiki_documenttag
+-- wiki_documentzone
+-- wiki_editortoolbar
+-- wiki_helpfulvote
+-- wiki_localizationtag
+-- wiki_localizationtaggedrevision
+-- wiki_reviewtag
+-- wiki_reviewtaggedrevision
+-- wiki_revision
+-- wiki_taggeddocument
+
 TRUNCATE account_emailconfirmation;
```

Comment:
This is interesting - EmailConfirmation is part of django-allauth, and is present in my production DB dump.  Maybe the user_profiles step caused it to fail.